Neos.Flow/Classes/Security/AccountIdentifier.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Security;
+
```

Comment:
A (short) doc comment describing what an `AccountIdentifier` is and does would be good.
Also we might consider marking it with
```
@Flow\Proxy(false)
```
to prevent the creation of a proxy class (until #1539 is resolved).

And, lastly, it would be a nice measure to implement the `\JsonSerializable` interface here and then have
```php
    public function jsonSerialize(): string
    {
        return $this->identifier;
    }
```
so that this behaves correct when being serialized

Reply:
(most of the comments apply to the other new value objects, too)